File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1115.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while(true){
    ns = lines.shift().split(' ')
    x = parseInt(ns[0])
    y = parseInt(ns[1])
    if(x==0 || y==0){
        break;
    }
    if(x>0){
        if(y>0){
    		console.log("primeiro");
        } else{
    		console.log("quarto");
        }
    } else{
        if(y>0){
    		console.log("segundo");
        } else{
            console.log("terceiro");
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1115.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        int x, y;
        while(true){
            x = ler.nextInt();
    		y = ler.nextInt();
            if(x==0 || y==0){
                break;
            }
            if(x>0){
                if(y>0){
    				System.out.println("primeiro");
                } else{
    				System.out.println("quarto");
                }
            } else{
                if(y>0){
    				System.out.println("segundo");
                } else{
                    System.out.println("terceiro");
                }
            }
        }
       
    }
}